lib/components/artist/biography.js

```diff
@@ -21,7 +21,7 @@ class Biography extends React.Component {
 
   render() {
     const artist = this.props.artist;
-    if (artist.blurb.length === 0 && artist.bio.length === 0) { return null; }
+    if (!artist.blurb && !artist.bio) { return null; }
```

Comment:
could you do `artist.blurb?.length` to account for both null & zero length? I think that's legit ES6